src/Lykke.Job.CandlesHistoryWriter.Services/Candles/RedisCacheCaretaker.cs

```diff
@@ -65,7 +65,7 @@ private async Task TruncateCacheAsync()
             {
                 foreach (var priceType in Constants.StoredPriceTypes)
                 {
-                    foreach (var timeInterval in Constants.DbStoredIntervals)
+                    foreach (var timeInterval in Constants.InitFromDbIntervals)
```

Comment:
RedisIntervals could be used here and code below can be made more straightforward, without inner loop. Optional.